.github/BOTMETA.yml

```
@@ -259,7 +259,7 @@ files:
   $modules/cloud/docker/docker_container.py:
     authors: ThomasSteinbach chouseknecht cove dusdanig felixfontein joshuaconner kassiansun softzilla zfil
```

@gundalow should this also be changed to:
```suggestion
    authors: ThomasSteinbach chouseknecht cove dusdanig felixfontein kassiansun softzilla zfil
```
? After all, the "real" author information (for docs etc.) is still in the module.